scripts/rollup/bundles.js

```diff
@@ -157,7 +157,7 @@ const bundles = [
     ],
     fbEntry: 'src/renderers/dom/shared/ReactDOMUnstableNativeDependenciesEntry',
     hasteName: 'ReactDOMUnstableNativeDependencies',
-    moduleType: RENDERER,
+    moduleType: ISOMORPHIC,
```

Comment:
This doesn't sound right.

Can we instead fix it to be able to safely use `ReactInternals`?